components/TravelMethodButtons/TravelMethodButtons.jsx

```diff
@@ -0,0 +1,50 @@
+import { Box, Button, Text, SimpleGrid, Icon, Flex } from '@chakra-ui/react';
+
+export function TravelMethodButtons({
+  transportIcon,
+  modesOfTransport,
+  handleTransportMode,
+  transportMode,
+}) {
+  return (
+    <SimpleGrid
+      columns={3}
+      defaultValue={transportMode}
```

Comment:
Why is there a need for a 'default' value for transport method?
Unless you have a good reason why we need one, I suggest removing it.

```suggestion
```

Reply:
This line is removed.